Rubberduck.Core/UI/Inspections/GroupingGridLocationConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.UI.Inspections
+{
+    class GroupingGridLocationConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is QualifiedModuleName qualifiedModuleName)
+            {
+                return $"{qualifiedModuleName} - {qualifiedModuleName.ComponentType}";
```

Comment:
The `ComponentType` is an enum, correct? The string should be localized then. Let's add `ComponentType_EnumMemberName` resource keys, so that we can have "Standard Module" (and "Module Standard" in French, but "Standardmodul" (?) in German) instead of a PascalCase "StandardModule" in all languages.